components/system/components/CodeBlock.js

```diff
@@ -4,21 +4,63 @@ import * as Constants from "~/common/constants";
 import { css } from "@emotion/react";
 
 const STYLES_CODE_BLOCK = css`
-  font-family: ${Constants.system.mono};
-  background-color: ${Constants.system.black};
-  color: ${Constants.system.white};
-  border-radius: 4px;
-  padding: 24px;
-  font-size: 14px;
-  word-wrap: break-word;
+  background-color: ${Constants.system.pitchBlack};
+  color: ${Constants.system.white}:
+  font-size: 16px;
+  border-color: ${Constants.system.yellow};
+  box-shadow: 0 2px 4px 0 rgba(0,0,0,0.50);
+  border-radius: 7px;
+  padding: 10px 10px;
+`;
+
+const STYLES_PADDING = css`
+  display:flex;
+  border-radius: 7px;
+  align-items: flex-start;
+  justify-content: space-between;
   white-space: pre-wrap;
-  width: 100%;
 `;
 
-export const CodeBlock = (props) => {
-  return (
-    <div css={STYLES_CODE_BLOCK}>
-      <code {...props} />
-    </div>
-  );
-};
+const STYLES_PRE = css`
+  color: ${Constants.system.darkGray};
+  font-family: ${Constants.font.monoCode};
+  min-width:25px;
+  padding: 3px 3px ;
+  flex: 0 ;
+`;
+
+const STYLES_CODE = css`
+  background-color: ${Constants.system.pitchBlack};
+  font-family: ${Constants.font.monoCode};
+
+  color: ${Constants.system.gray};
+  text-align: left;
+  padding: 3px 3px;
+  flex: 10 ;
```

Comment:
what is this for? this should be `min-width: 10%; width: 100%;` for a fluid container. and the `text-align: left;` is not necessary.